README.md

```diff
@@ -8,21 +8,25 @@
 
 The AWS SDK for iOS provides a library and documentation for developers to build connected mobile applications using AWS.
 
+We recommend using the latest v2 version of AWS Amplify Library for Swift to quickly implement common app use cases like Authentication, Storage, Push Notifications and more that follow patterns idiomatic to Swift like async/await. 
+
+You can click into documentation for the different features that Amplify supports below or head to [the Amplify Library for Swift documentation](https://docs.amplify.aws/lib/auth/getting-started/q/platform/ios/) to learn more about all the features you have access to. You can also use AWS Amplify with [your existing AWS cloud resources](https://docs.amplify.aws/lib/project-setup/use-existing-resources/q/platform/ios/) or access the underlying SDK via the Escape Hatch from AWS Amplify. 
```

Comment:
would you want to give some context on the Amplify SDKs being built on top of the AWS SDK before mentioning the escape hatch?

Also - the first sentence is a bit confusing, head into the documentation from below, or click into this main link - is there a specific thing you want to push users to?

Reply:
Good point, I reordered the information to offer better context. Please review now.